### ISSUE TITLE
brittany pretty printer returned syntactically invalid result.

### ISSUE THREAD

**Issue by lpil** [lspitzner/brittany#89]
Hello!

I've found some Haskell code that Brittany doesn't know how to format:
 
```haskell
type instance AuthServerData (AuthProtect "JWT") = DBUser
```

Trying to format this returns this error in the command line 

```
ERROR: brittany pretty printer returned syntactically invalid result.
```

Thanks!

**Comment by lspitzner** [comment]
Yeah this is https://github.com/alanz/ghc-exactprint/issues/52. It requires either a fix upstream or `brittany` must learn to layout `type instance`s itself.

**Comment by lpil** [comment]
Thanks for the update :)

By the way, thank you so much for making this tool. I've only just started using it but I feel the output format is very nice :(

**Comment by lspitzner** [comment]
I noticed this bug cannot be prevented by using inline-config (in general i was hoping that `-- brittany-disable-next-binding` would help work around any unimplemented stuff), so instead I implemented a workaround for this in e9aacb27ffcd1bdf350d59a56178688284cd4f8f.

**Comment by lspitzner** [comment]
fixed in 0.11.0.0
